### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by zigaLuksic**
There is actually a lot less work than I feared there would be

### CODE REVIEW

File: examples/core/CoreOverview.ipynb [hunk 1]
```diff
@@ -793,11 +843,18 @@
...
+       "model_id": "a0e29f064af545a79ee277e719c27391",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0%|          | 0/5 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
```
Comment (zigaLuksic) [inline examples/core/CoreOverview.ipynb:857]:
```suggestion
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 5/5 [00:00<00:00, 510.50it/s]\n"
     ]
```

File: examples/visualization/EOPatchVisualization.ipynb [hunk 2]
```diff
@@ -371,7 +342,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.4"
+   "version": "3.8.10"
```
Comment (zigaLuksic) [inline examples/visualization/EOPatchVisualization.ipynb:345]:
```suggestion
   "version": "3.10.4"
```

**Review by zigaLuksic — COMMENTED** [review]
just reverting some things, vscode can be a PITA for this sometimes

**Review by mlubej — APPROVED** [review]
Made minor edits, otherwise looked only at the changes, so I'm gonna trust you with the unchanged content, unless you think the examples would benefit from a second comb-through

**Comment by zigaLuksic** [comment]
> Made minor edits, otherwise looked only at the changes, so I'm gonna trust you with the unchanged content, unless you think the examples would benefit from a second comb-through

Thanks. No need for second passes IMO, i ran the notebooks and adjusted fields that produced warnings. If we missed something the users will tell us.

**Pull request merged**